File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;
import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

import java.util.List;
import java.util.Map;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface BrandService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbBrand> findAll();

	PageResult findPage(int page, int rows);

    void save(TbBrand tbBrand);

	void deleteBrand(Long[] id);

	TbBrand findOne(Long id);

	PageResult search(TbBrand tbBrand, int page, int rows);

    List<Map> getBrandList();
}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {
    this.deleteBrand = function(selectIds){
        return $http.post('../brand/deleteBrand.do',selectIds);
    }
    this.findOne = function (id) {
        return $http.get('../brand/findOne.do?id='+id);
    }
    this.save = function (entity) {
        return $http.post('../brand/save.do',entity);
    }
    this.findAll = function () {
        return $http.get('../brand/findAll.do');
    }
    this.search = function (searchEntity,page,rows) {
        return $http.post("../brand/search.do?page="+page+"&rows="+rows,searchEntity);
    }
    this.findPage = function (page,rows) {
        return $http.get("../brand/findPage.do?page="+page+"&rows="+rows);
    }
    this.getBrandList=function () {
        return $http.get('../brand/getBrandList.do');
    }
});